recipes-app/iot2050-module-firmware-update/iot2050-module-firmware-update_01.bb

```diff
@@ -0,0 +1,26 @@
+#
+# Copyright (c) Siemens AG, 2024
+#
+# Authors:
+#  Li Hua Qian <huaqian.li@siemens.com>
+#
+# This file is subject to the terms and conditions of the MIT License.  See
+# COPYING.MIT file in the top-level directory.
+#
+
+DESCRIPTION = "Customized Module(Special DI) Firmware Update Scripts"
```

Comment:
At what is "Special DI" now?

Reply:
@huaqianli: 

IOT2050 Customized external signal module, such as `SM SENS DI`. This kind of module is not PLC1200 standard, but "customized" by IOT2050. Hence such module could not be upgraded within the PLC ecosystem, but we have to provide method to support the FW upgrading for them.